specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/azureFirewall.json

```diff
@@ -192,6 +192,58 @@
         "x-ms-long-running-operation-options": {
           "final-state-via": "azure-async-operation"
         }
+      },
+      "patch": {
+        "tags": [
+          "AzureFirewalls"
+        ],
+        "operationId": "AzureFirewalls_UpdateTags",
+        "description": "Updates tags for an Azure Firewall resource.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "azureFirewallName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure Firewall."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AzureFirewall"
+            },
+            "description": "Parameters supplied to the create or update Azure Firewall operation."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting AzureFirewall resource.",
+            "schema": {
+              "$ref": "#/definitions/AzureFirewall"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update Azure Firewall Tags": {
+            "$ref": "./examples/AzureFirewallUpdateTags.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
Per specs, operation should be marked as long-running if it's returning codes 201 or 202.
https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation
I've tried various PATCHes, including Azure Firewall, all of them returned 200 and updated resources synchronously, so I don't think this option is needed here

Reply:
Thank you! You're right. I updated the spec.